### PR TITLE
Fix some typos and add to spell-check

### PR DESCRIPTION
**Pull request by DelazJ**
Question: is the header of these files normal?

### CODE REVIEW

**Comment by DelazJ** [comment]
I can't understand what's going wrong with Travis here

File: src/core/simplify/effectivearea.h [hunk 1]
```diff
@@ -68,7 +68,7 @@ struct MINHEAP
 };
 
 /**
- * Structure to hold pointarray and it's arealist.
+ * Structure to hold pointarray and its arealist.
```
Comment (m-kuhn) [inline src/core/simplify/effectivearea.h]: Make that a `point array` as well, I guess.

**Review by m-kuhn — APPROVED** [review]

File: src/core/simplify/effectivearea.h (hunk 1 above)
Comment (DelazJ) [reply src/core/simplify/effectivearea.h]: I thought it was a variable but seems not. Should `arealist` also become `area list` (not sure what it's in the code)?

**Comment by DelazJ** [comment]
@m-kuhn no idea about the header? It's all about PostGIS.

**Comment by m-kuhn** [comment]
@DelazJ what about the header?

**Comment by DelazJ** [comment]
I meant the (usual) license section at the top of the file https://github.com/DelazJ/QGIS/blob/727514d9af57ddb4ffb19691beb5944cad74f14d/src/core/simplify/effectivearea.cpp#L1 Does it correspond to the content?

**Comment by m-kuhn** [comment]
I guess so, unless someone offers a good reason to change this, I'd say keep it to track origin. But I am not a lawyer.

**Comment by DelazJ** [comment]
Then all is good... ;)

**Pull request merged**